scripts/go/configs/golangci.yml

```diff
@@ -37,19 +37,59 @@ linters:
     - varcheck
     - whitespace
 
-linter-settings:
+linters-settings:
```

Comment:
this either change in some version of golangci-lint, or it never worked in the first place